[Content] Remnant Teciimach Torpedo (#6707)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

			choice
				`	"Thanks for the explanation."`
					goto empno
				`	"What does 'Warren' mean?"`
			`	"It is our term for the network of caves, tunnels, and spaces deep underground. All three of our main worlds have them in some form or another. We know a little about the original people who built them, but not nearly as much as we would like. They called themselves the Zitchal Maugdrakoi, and one of the most distinguishing features we have been able to learn about them is that they do not seem to attach much importance to directionality. For example, in a control room we found, there were instruments and controls on every available surface, with just a scattering of handholds scattered around to support the operator." She shrugs. "Plume or Dusk would know more than I about them. I mostly focus on understanding their equipment."`
			label empno
			`	"Well, I should get back to work, then. Next time you are in space, take a look at how our ships fare in battle. Hopefully you will see an improvement." She grabs a couple components from a nearby lift, and scrambles back up the side of the Merganser to continue working on an EMP bay.`
				accept

> 	npc

Reviewer: they did not fire a single EMP torpedo, maybe make sure to spawn those vairants?

Author: I had, for some reason, totally missed putting EMP pods on that Merganser variant... Thanks for pointing it out!

			label empno
			`	"Well, I should get back to work, then. Next time you are in space, take a look at how our ships fare in battle. Hopefully you will see an improvement." She grabs a couple components from a nearby lift, and scrambles back up the side of the Merganser to continue working on an a new bay.`
				accept
	npc
		government "Remnant"
		personality staying uninterested disables plunders
		fleet
			names "remnant"
			variant
				"Merganser II" 3
	npc
		government "Remnant"
		personality staying uninterested disables plunders
		fleet "Small Remnant"
	npc evade
		government "Korath"

> 		personality heroic plunders harvests target

Author: That's a good suggestion. Implemented. Thanks!

			label mainreturn
			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles mischievously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`

mission "Remnant: Teciimach Deployment"

> 	name "New EMP deployment"
> 	description "The Remnant are deploying a new version of the EMP, and invite you to watch."
> 	invisible
> 	minor
> 	landing

Reviewer 2: ```suggestion
	minor
	landing
	name "New EMP deployment"
	description "The Remnant are deploying a new version of the EMP, and invite you to watch."
	invisible
```

> 	minor

Reviewer 3: I think this should go
```
invisible
minor
landing
```
Based in the creating missions wiki page.

	source
		attributes "shipyard" "remnant"
	to offer
		has "event: remnant: cognizance calm"

> 		has "event: remnant research update bjd1"

Author: I'll go double check this and make the change. Thanks!

> 		has "event: remnant research update bjd1"

Author: Done. And this is a good thing, too, since I'll probably have more stuff tying back to those missions in the span of the next chapter or two.

> 		has "event remnant research update bjd1"

Author: I would like to, yes, but was concerned about breaking saves and whatnot. Was considering doing that change as a standalone PR.

	on offer
		event "remnant: teciimach deployment"
		conversation
			`You note a disturbance in the shipyards before the <ship> even touches the ground. Every pad is filled with ships, camels pulling wagon trains loaded with canisters trundle out of the outfitter heading for ships, and technicians clamber everywhere. As you step out of your ship, Taely slides down the side of a Merganser resting next to you.`
			`	"Good timing, Captain! We just rolled out an upgrade to some of our oldest equipment, and our Mergansers finally have something to carry as first-strike weapon. Interested in taking a look?"`
			choice
				`	"Yes"`

> 				`	"No"`

Reviewer 4: The negative option is usually less blunt than this.
```suggestion
				`	"Sorry, but I'm a bit busy at the moment."`
```

					goto empno
			`	She turns and heads towards a nearby lift where another crew of Remnant had just moved several EMP torpedo canisters into place. "Well, you have probably seen us use EMP torpedoes now and then, right? Well, as you probably noticed, they have a few tactical issues. They are slow, they often hit our ships almost as much as our targets, and they take up space we could otherwise be using for more important things like laboratories or scanners." She taps a few commands into the console, and the lift begins raising the canisters towards the waiting ship.`

> 			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focusing on the torpedoes themselves, others focusing on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole works. Very impressive work, and the results should be in the outfitter for you in the next day or two."`

Reviewer 4: ```suggestion
			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole works. Very impressive work, and the results should be in the outfitter for you in the next day or two."`
```

> 			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole structure. Very impressive work, and the results should be in the outfitter for you in the next day or two."`

Reviewer 2: One last slight change in how Taely describes the changes made to the weapon to account for the suggested stats change to lower the capacity to 6.
```suggestion
			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole structure. Unfortunately it carries a few less torpedoes, but that is more than made up for by the improvements to the munitions, which are better than their predecessor in almost every way. Very impressive work, and the results should be in the outfitter for you in the next day or two."`
```

> 			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole structure. Unfortunately it carries a few less torpedoes, but that is more than made up for by the improvements to the munitions, which are better than their predecessor in almost every way. Very impressive work, and the results should be in the outfitter for you in the next day or two."`

Reviewer 2: Heads up that the suggestion I made to this line assumed the other comment saying we could make the bay's mass 7 and capacity 6.

Author: Fixed.

> 			`	She looks at you and nods contemplatively. "One thing worth noting: You are making great strides in learning our language, so new items that appear in our outfitter will be listed under their proper name instead of substitutes from the ancestral tongue."`

Reviewer 4: ```suggestion
			`	She looks at you and nods contemplatively. "One thing worth noting - you are making great strides in learning our language, so new items that appear in our outfitter will be listed under their proper name instead of substitutes from the ancestral tongue."`
```

			label empno
			`	"Well, I should get back to work, then. Next time you are in space, take a look at how our ships fare in battle. Hopefully you will see an improvement." She grabs a couple components from a nearby lift, and scrambles back up the side of the Merganser to continue working on an a new bay.`
				accept
	npc
		government "Remnant"
		personality staying uninterested disables plunders
		fleet
			names "remnant"
			variant
				"Merganser (II)" 3
	npc
		government "Remnant"
		personality staying uninterested disables plunders
		fleet "Small Remnant"
	npc evade
		government "Korath"
		personality heroic plunders harvests staying
		fleet "Korath Ember Waste Raid"
	on complete
		dialog

> 			`You notice the most recent Korath raiders have been defeated and decide to check your sensor logs to see how the new EMP torpedoes Taely talked about performed. You note that the EMP bursts no longer have an impact on the Remnant ships. This could make them much more versatile.`

Reviewer 3: It occurs to me that this is poor wording jf the player decided to get involved in the fight.

```suggestion
			`Now that the most recent Korath raiders are defeated, you decide to check your sensor logs to see how the new EMP torpedoes Taely talked about performed. You note that the EMP bursts no longer have an impact on the Remnant ships. This could make them much more versatile.`
```

> 			`Now that the most recent Korath raiders have been defeated, you decide to check your sensor logs to see how the new EMP torpedoes Taely talked about performed. You note that the EMP bursts no longer have an impact on the Remnant ships. This could make them much more versatile.`

Reviewer 2: Since "safe" affects all friendly ships and not just the Remnant's ships, and the description was changed to say it's a shaped charge that avoids friendlies, should this line be updated? I'm sure the player would notice their own ship(s) not getting hit as well.
```suggestion
			`Now that the most recent Korath raiders have been defeated, you decide to check your sensor logs to see how the new EMP torpedoes Taely talked about performed. You note that the EMP bursts no longer have an impact on friendly ships. This could make them much more versatile.`
```
